src/ApparentHorizons/StrahlkorperGr.hpp

```diff
@@ -477,4 +480,23 @@ double irreducible_mass(double area);
  */
 double christodoulou_mass(double dimensionful_spin_magnitude,
                           double irreducible_mass);
+
+/*!
+ * \ingroup SurfacesGroup
+ * \brief Dimensionless spin magnitude of a `Strahlkorper`.
+ *
+ * \details
+ * This function computes the dimensionless spin magnitude from the
+ * dimensionful spin magnitude \f$S\f$ and the christodoulou mass \f$M\f$
+ * of a black hole. Specifically, computes
+ * \f$\frac{S}{M^2}\f$.
+ */
+
+void dimensionless_spin_magnitude(const gsl::not_null<double*> result,
+                                  const double dimensionful_spin_magnitude,
+                                  const double christodoulou_mass);
+
+double dimensionless_spin_magnitude(const double dimensionful_spin_magnitude,
+                                    const double christodoulou_mass);
+
```

Comment:
While squashing can you add `/// @}` here? This makes the dox comment aplly to both functions

Reply:
yep , can do, making the changes now